### ISSUE TITLE
puppet_webhook is not compatible with Ruby < 2.2.2

### ISSUE THREAD

**Issue by dhollinger** [voxpupuli/puppet_webhook#56]
When attempting to install `puppet_webhook` on Ruby < 2.2.2, it will error out with the following error:

```
ERROR:  Error installing puppet_webhook:
	rack requires Ruby version >= 2.2.2.
```

This has potential impact to any `puppet_webhook` user that uses Puppet < 5.0.0. We either need to update the gemspec and code to make it compatible with Sinatra < 2.x or remove information listing compatibility with Ruby < 2.2.2

**Comment by bastelfreak** [comment]
I am fine with dropping puppet 4. This is a nee project and puppet 4 is EOL in the end of this year anyways.

**Comment by dhollinger** [comment]
Solved by #58
